pkg/image/apis/image/types.go

```diff
@@ -28,6 +28,15 @@ const (
 	// ExcludeImageSecretAnnotation indicates that a secret should not be returned by imagestream/secrets.
 	ExcludeImageSecretAnnotation = "openshift.io/image.excludeSecret"
 
+	// DockerImageLayersOrderAnnotation describes layers order in the docker image.
+	DockerImageLayersOrderAnnotation = "openshift.io/image.dockerLayersOrder"
+
+	// DockerImageLayersOrderAscending means that an image layers in the ascending order.
+	DockerImageLayersOrderAscending = "asc"
+
+	// DockerImageLayersOrderDescending means that an image layers in the descending order.
```

Comment:
indicates that layers are sorted in reversed order of their addition (from newest to oldest).